### ISSUE TITLE
Error when trying to lunch application that has mercury installed

### ISSUE THREAD

**Issue by kawzar** [postlight/mercury-parser#334]
Platform: 64-bit Windows 10
Mercury-Parser Version: "^2.0.0"
Node Version: v10.15.3

I have installed mercury-parser through npm in an angular application. When running the application, it fails when doing the npm start for the following: 
ERROR in ./node_modules/@postlight/mercury-parser/dist/mercury.js
Module not found: Error: Can't resolve 'cheerio' in 'C:\Projects\furry-garbanzo\link-manager\node_modules\@postlight\mercury-parser\dist'
ERROR in ./node_modules/@postlight/mercury-parser/dist/mercury.js
Module not found: Error: Can't resolve 'iconv-lite' in 'C:\Projects\furry-garbanzo\link-manager\node_modules\@postlight\mercury-parser\dist'

I have tried: installing cheerio & iconv-lite both in the application and globally, but it fails still. Below is the log:

 ```
0 info it worked if it ends with ok
1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'start' ]
2 info using npm@6.4.1
3 info using node@v10.15.3
4 verbose run-script [ 'prestart', 'start', 'poststart' ]
5 info lifecycle link-manager@0.0.0~prestart: link-manager@0.0.0
6 info lifecycle link-manager@0.0.0~start: link-manager@0.0.0
7 verbose lifecycle link-manager@0.0.0~start: unsafe-perm in lifecycle true
8 verbose lifecycle link-manager@0.0.0~start: PATH: C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin;C:\Projects\furry-garbanzo\link-manager\node_modules\.bin;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Users\mhaspert\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0;C:\Windows\System32\OpenSSH;C:\Program Files (x86)\AMD\ATI.ACE\Core-Static;C:\Program Files\dotnet;C:\Program Files\Microsoft SQL Server\130\Tools\Binn;C:\Program Files (x86)\Microsoft SQL Server\Client SDK\ODBC\130\Tools\Binn;C:\Program Files (x86)\Microsoft SQL Server\140\Tools\Binn;C:\Program Files (x86)\Microsoft SQL Server\140\DTS\Binn;C:\Program Files (x86)\Microsoft SQL Server\140\Tools\Binn\ManagementStudio;C:\Program Files\Microsoft VS Code\bin;C:\Program Files (x86)\ATI Technologies\ATI.ACE\Core-Static;C:\Program Files\Git\cmd;C:\Program Files\nodejs;C:\Users\mhaspert\AppData\Local\Microsoft\WindowsApps;C:\Users\mhaspert\AppData\Roaming\npm
9 verbose lifecycle link-manager@0.0.0~start: CWD: C:\Projects\furry-garbanzo\link-manager
10 silly lifecycle link-manager@0.0.0~start: Args: [ '/d /s /c', 'ng build && node ./server/app.js' ]
11 silly lifecycle link-manager@0.0.0~start: Returned: code: 1  signal: null
12 info lifecycle link-manager@0.0.0~start: Failed to exec start script
13 verbose stack Error: link-manager@0.0.0 start: `ng build && node ./server/app.js`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\index.js:301:16)
13 verbose stack     at EventEmitter.emit (events.js:189:13)
13 verbose stack     at ChildProcess.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\lib\spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:189:13)
13 verbose stack     at maybeClose (internal/child_process.js:970:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
14 verbose pkgid link-manager@0.0.0
15 verbose cwd C:\Projects\furry-garbanzo\link-manager
16 verbose Windows_NT 10.0.17134
17 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "start"
18 verbose node v10.15.3
19 verbose npm  v6.4.1
20 error code ELIFECYCLE
21 error errno 1
22 error link-manager@0.0.0 start: `ng build && node ./server/app.js`
22 error Exit status 1
23 error Failed at the link-manager@0.0.0 start script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]

```
## Expected Behavior

The application runs properly and the dependencies of mercury-parser work

Thank you in advance

**Comment by DrLuthor** [comment]
Same problem here

Platform: 64-bit macOS
Mercury-Parser Version: "^2.0.0"
Node: 11.13.0
Rails : 5.2.1.1
Ruby: 2.5.1p57
Postgres: 11.2

**Comment by toufic-m** [comment]
This has been fixed in #302 and will be included in the next release/package version.

**Comment by toufic-m** [comment]
`v.2.1.0` (ca47f9c) has been released. Please update the `@postlight/mercury-parser` package to get the latest fixes.

https://github.com/postlight/mercury-parser/blob/master/CHANGELOG.md#210-apr-10-2019
